solutions/landing-zone-v2/namespaces/hierarchy.yaml

```diff
@@ -112,5 +112,41 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 subjects:
   - name: cnrm-controller-manager-policies
+    namespace: cnrm-system
+    kind: ServiceAccount
+---
+# IAM Permissions to create folder sinks
+# Grant viewer role on the hierarchy namespace to config-control K8S SA
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata: # kpt-merge: hierarchy/allow-hierarchy-resource-reference-from-config-control
+  name: allow-hierarchy-resource-reference-from-config-control
+  namespace: hierarchy
+  annotations:
+    cnrm.cloud.google.com/ignore-clusterless: "true"
+    internal.kpt.dev/upstream-identifier: 'rbac.authorization.k8s.io|RoleBinding|hierarchy|allow-hierarchy-resource-reference-from-config-control'
```

Comment:
the `internal.kpt.dev/upstream-identifier` annotation should be removed

Reply:
Fixed, please resolve the comment if satisfied.